algorithms/Networks/Hamming_Code/README.md

```diff
@@ -0,0 +1,10 @@
+# Hamming Code
```

Comment:
Remove this file.

To add docs, check this repo: https://github.com/AllAlgorithms/algorithms

Reply:
Ok, no problem. I have removed the README.md file for this section.